static/js/groups_page.js

```diff
@@ -411,6 +484,147 @@ var groupPage = {
+            // str += '<button onclick = "groupPage.SharedDiscussionsTable.operationEvents(' + row.owner_email + ',' +row.id + ')">dfdff</button>';
+            str +='<div><h6><i>' + '#'+ row.id + groupPage.SharedDiscussionsTable.dateFormatter(value, row) + ' by ' + row.owner_email +  '</i></h6></div>';
+            str +='</div>';
+            
+            return str;
+        },
+        visibilityFormatter: function (value, row) {
+            if (row.is_resolved === 1) {
+                return '<i class="fa fa-lock discussionLock' + String(row.id) + '"></i><i class="fa fa-unlock passive discussionUnlock' + String(row.id) + '"></i> ';
+            } else {
+                return '<i class="fa fa-unlock discussionUnlock' + String(row.id) + '"></i><i class="fa fa-lock passive discussionLock' + String(row.id) + '"></i> ';
+            }
+        },
+        dateFormatter: function(value, row){
+            if(row.is_resolved ===1){
+                var str = '';
+                str += '<span class="discussionLock' + String(row.id) + '">';
+                str += ' closed ';
+                str += moment(row.updated_at).fromNow();
+                str += '</span>';
+                str += '<span class="passive discussionUnlock' + String(row.id) + '">';
+                str += ' opened ';
+                str += moment(row.created_at).fromNow();
+                str += '</span>';
+                return str;
+            } else{
+                var str = '';
+                str += '<span class="discussionUnlock' + String(row.id) + '">';
+                str += ' opened ';
+                str += moment(row.created_at).fromNow();
+                str += '</span>';
+                str += '<span class="passive discussionLock' + String(row.id) + '">';
+                str += ' closed ';
+                str += moment(row.updated_at).fromNow();
+                str += '</span>';
+                return str;
+            }
+        },
+        nameFormatter: function (value, row) {
+            console.log(row);
+            return $('<a>').attr('href', $('#GroupID').val() +'/discussions/' + row.id).text(row.topic)[0].outerHTML;
+        },
+        deleteFormatter: function (value, row) {
+            if(row.owner_email === $('#UserEmail').val()){
+                return [
+                '<a style = "float:right;"  onclick = "groupPage.SharedDiscussionsTable.operationEvents(\''+row.owner_email+'\',\''+row.id+'\')"   id="deleteModal' + String(row.id) + '" href="javascript:void(0)" title="Delete">',
+                'Delete <i class="fa fa-trash"></i>&nbsp;',
+                '</a>'
+            ].join('');
+            }
+            else return '';
+        },
+        resolveFormatter: function (value, row) {
```

Comment:
change the name of the method to reflect the change in name of the attribute --> "isClosedFormatter"